src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,35 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Task type event.
+ */
+public class Event extends Task {
+
+    private final LocalDateTime at;
+    private final DateTimeFormatter formatter =
+            DateTimeFormatter.ofPattern("MMM dd yyyy, HH:mm");
+
+    public Event(String description, LocalDateTime at) {
+        super(description);
+        this.at = at;
+    }
+
+    @Override
```

Comment:
Good usage of @Override which explicitly labels methods that overrides parent classes.

Reply:
Nice